src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -20,6 +20,7 @@ import * as ReportActionContextMenu from './ContextMenu/ReportActionContextMenu'
 import VirtualKeyboard from '../../../libs/VirtualKeyboard';
 import * as EmojiUtils from '../../../libs/EmojiUtils';
 import reportPropTypes from '../../reportPropTypes';
+import addEmojiToComposer from '../../../libs/addEmojiToComposer';
```

Comment:
Should this component be converted to PureComponent? It is 75% the same as ReportActionCompose.

Reply:
hm, in this component I see only one check that could be omitted by `PureComponent`. In this case, I think this one check is easier + costs fewer resources than shallow comparing _all_ props as PureComponent would do.
So I feel fine with leaving it as a regular component.